src/PctvData.cpp

```diff
@@ -231,6 +230,29 @@ bool Pctv::LoadChannels()
   return true;  
 }
 
+PVR_ERROR Pctv::GetChannelStreamProperties(const PVR_CHANNEL* channel, PVR_NAMED_VALUE* properties, unsigned int* iPropertiesCount)
+{
+  std:string strUrl = "";
+  for (unsigned int i = 0; i<m_channels.size(); i++)
```

Comment:
Could you please insert blank around the `<`?

Reply:
You also might want to look at my comments re. string initialization and 'better' for loop syntax in your other (pvr.filmon) PR. ;-)